maps/src/test/java/org/odk/collect/maps/layers/DirectoryReferenceLayerRepositoryTest.kt

```diff
@@ -176,6 +177,23 @@ class DirectoryReferenceLayerRepositoryTest {
         assertThat(projectLayersDir.listFiles()[0].readText(), equalTo("blah"))
     }
 
+    @Test
+    fun delete_returnsLayerWithCorrectName() {
```

Comment:
Should this `delete_deletesLayerWithId`? I don't understand `returnsLayerWithCorrectName` in this case.

Reply:
Right I copied one of the tests and didn't update the name properly.